sources/packages/api/src/testHelpers/controllers/auth-test/auth-test.controller.ts

```diff
@@ -42,7 +42,7 @@ export class AuthTestController {
   /**
    * Only authenticated users with specific role will have access to this endpoint.
    */
-  @Roles(Role.Student)
+  @Roles(Role.AESTReports)
```

Comment:
I know that this is test controller. But why did we use AESTReports role to authorize this?

Reply:
as you said. its test controller. we are testing a role for a aest user